app/views/shared/_search_form.html.erb

```diff
@@ -1,3 +1,30 @@
+
```

Comment:
The advanced search just used the markup from the upper right, so the new user dropdown is duplicated and I don't think it should be. 

<img width="815" alt="screen shot 2017-07-27 at 8 56 34 am" src="https://user-images.githubusercontent.com/174744/28671130-9112693e-72a9-11e7-84eb-881b5a3a20b4.png">